src/runtime/link-accounts-flow.js

```diff
@@ -206,8 +213,11 @@ export class LinkSaveFlow {
     /** @private @const {!../components/dialog-manager.DialogManager} */
     this.dialogManager_ = deps.dialogManager();
 
-    /** @private {!../api/subscriptions.SaveSubscriptionRequest} */
-    this.request_ = request;
+    /** @private {!saveRequestCallback} */
+    this.callback_ = callback;
+
+    /** {?Promise<../api/subscriptions.SaveSubscriptionRequest>} */
```

Comment:
`@private` and the `SaveSubscriptionRequest` must be `!`. And further, we is the `SaveSubscriptionRequest` in the `@callback` definition is used w/o namespace and with namespace here. Does compiler give any warnings about this?

Reply:
done